src/cmd-build

```diff
@@ -255,7 +263,15 @@ EOF
 if [ -d "${workdir}/src/yumrepos" ]; then
     prepare_git_artifacts "${workdir}/src/yumrepos" "${PWD}/coreos-assembler-yumrepos-git.json"
 fi
-prepare_git_artifacts "${configdir_gitrepo}" "${PWD}/coreos-assembler-config-git.json" "${PWD}/coreos-assembler-config.tar.gz"
+
+# set the config repo archive file name
+config_archive="${PWD}/coreos-assembler-config.tar.gz"
+if ! ${CONFIG_ARCHIVE} >/dev/null; then
```

Comment:
I always find it a bit weird to have variables that evaluate to the `true`/`false` binaries; a lot of the rest of the code here would use e.g. `if test "${CONFIG_ARCHIVE}" = "true"; then ...`